src/style.css

```diff
@@ -0,0 +1,22 @@
+
+body {
+  font-family: Arial;
+  font-size: 13px;
+  line-height: normal;
+}
+
+#h1-heading {
```

Comment:
do not use `id` to style elements. For this use `class`

Reply:
done